helm/kanister-operator/values.yaml

```diff
@@ -51,6 +51,18 @@ repositoryServerController:
   logLevel: 'info'
   container:
     name: 'repository-server-controller'
+  service:
+    # port is used as the secured service port if the validating
+    # webhook is enabled.
+    port: 444
+  # This flag is used to enable validating webhook for repository server CR
+  # The TLS certificates for blueprint validating webhook server and 
+  # repositoryserver validing webhook server are same and can be provided
+  # under field `bpValidatingWebhook.tls` 
```

Comment:
As discussed this will break the the kanister upgrades, if we start using the new helm values, hence I will mention a comment on existing value `bpValidatingWebhook` that this certificate will be used for Repository Server CR validation as well